library/common-library/templates/_security-context.tpl

```diff
@@ -18,5 +25,12 @@
     {{- toYaml .Values.podSecurityContext -}}
 {{- else if $global.podSecurityContext -}}
     {{- toYaml $global.podSecurityContext -}}
+{{- else -}}
+    {{- include "common.securityContext.podDefaults" . -}}
```

Comment:
Oks, so in the values there will be all the block in podDefaults and only honoured if others not set, I like this solution. I think it's clear for the users looking at the values and covers the need we had.

Reply:
`values.yaml` should be empty for the `.global` to work but the default can change using other helpers like the `privileged` which I am waiting for this to be merged to start working on it.